### DISCUSSION TITLE
add `leading-0` utility

### DISCUSSION THREAD

**Discussion by MichaelAllenWarner** [tailwindlabs/tailwindcss#6029] [Ideas]
Occasionally there's reason to set `line-height: 0` (not for normal text, of course; rather, in "decorative" scenarios where you want to prevent vertical "protrusions"). Not difficult to add it to the TW config (as I do for every project) or use `leading-[0]`, but I think it would be a good addition to core.

**Comment by MichaelAllenWarner** [comment]
In the spirit of https://github.com/tailwindlabs/tailwindcss/pull/10294 (adding `delay-0` and `duration-0` utilities), how about we add a default `leading-0` utility? This one's even used in Tailwind's [preflight](https://github.com/tailwindlabs/tailwindcss/blob/master/src/css/preflight.css#L124-L130)!

**Comment by 52461** [comment]
Hello,
In economics, utility is a term used to determine the worth or value of a good or service .  [Home Bargains Portal
](https://www.homebargainsportal.net/)
